beast/examples/phat_small/datamodel.py

```diff
@@ -289,6 +287,5 @@ def get_obscat(obsfile=obsfile, distanceModulus=distanceModulus,
     obs: GenFluxCatalog
         observation catalog
     """
-    obs = GenFluxCatalog(obsfile, distanceModulus=distanceModulus,
-                         filters=filters)
+    obs = GenFluxCatalog(obsfile, filters=filters)
```

Comment:
distanceModulus has gone because it's not used or what?

Reply:
For completeness, we never covert the observed fluxes to intrinsic fluxes when doing the fitting.  Are you arguing for keeping this for some other purpose?  Asking to make sure.

Reply:
Sorry for my misunderstanding -- your thorough explanation below was very helpful. Removing distanceModulus here is OK, as I see no need/use for it.

Reply:
No worries.  Very useful conversation to have (and have archived!).